README.yaml

```
@@ -133,6 +133,30 @@ usage: |-
   }
   ```
 
+  Allowing specific principal ARNs to perform actions on the bucket:
+
+  ```hcl
+  module "s3_bucket" {
+    source = "cloudposse/s3-bucket/aws"
+    # Cloud Posse recommends pinning every module to a specific version
+    # version     = "x.x.x"
```

Our existing examples are inconsistent, but this is closer to standard Terraform formatting:
```suggestion
    # version = "x.x.x"
```

---

resolved